Makefile

```diff
@@ -131,7 +131,7 @@ ifeq ("$(TRAVIS_COVERAGE)", "1")
 	$(OVERALLS) -project=github.com/pingcap/tidb \
 			-covermode=count \
 			-ignore='.git,vendor,cmd,docs,LICENSES' \
-			-concurrency=2 \
+			-concurrency=4 \
```

Comment:
what does this change used for?

Reply:
Make the `idc-jenkins-ci-tidb/code_coverage` use less time.
And the side effect is it will take more memory.

https://github.com/go-playground/overalls
>   -concurrency
    Limit the number of packages being processed at one time.
    The minimum value must be 2 or more when set.
    example: -concurrency=5
    default: unlimited

-------
Before, Travis-CI only has about 7.5GB RAM, and we will make the concurrency limit in https://github.com/pingcap/tidb/pull/4535.
But not, Travis-CI has been deleted, and the code_coverage runs in our idc.
IDC has more than 100GB memory, I think we can make this field bigger.

Reply:
Before:
![image](https://user-images.githubusercontent.com/7782671/56480538-1abafe80-64ed-11e9-88d1-432babf9fcac.png)

After:
![image](https://user-images.githubusercontent.com/7782671/56480542-25759380-64ed-11e9-8508-6c0fd540ef18.png)

Reply:
@zhouqiang-cl Is it allowed that several `code_coverage` jobs can run in a same machine in IDC?